Data/HashMap/Internal.hs

```diff
@@ -2247,34 +2247,62 @@ clone ary =
 ------------------------------------------------------------------------
 -- Bit twiddling
 
+-- TODO: Name this 'bitsPerLevel'?! What's a "subkey"?
 bitsPerSubkey :: Int
 bitsPerSubkey = 5
 
+-- | The size of a 'Full' node, i.e. @2 ^ 'bitsPerSubkey'@.
 maxChildren :: Int
 maxChildren = 1 `unsafeShiftL` bitsPerSubkey
 
+-- | Bitmap with the lowest 'bitsPerSubkey' bits set, i.e. @0b11111@.
 subkeyMask :: Bitmap
```

Comment:
We never use this mask as a bitmap, so it should probably simply be a `Word` or maybe a new `Mask` type.

Reply:
Agreed.

Reply:
I realized that `subkeyMask` is used only once: in `index`. It's probably not worth adding a new type for it, so I'll go with `Word`.